src/lib.rs

```diff
@@ -79,7 +79,7 @@ impl Lnd {
             Lnd::connector(certificate_bytes).map_err(LndConnectError::Connector)?;
 
         let transport = tonic::transport::Endpoint::new(destination)
-            .and_then(move |d| d.connect_with_connector_lazy(https_connector))
+            .and_then(move |d| Ok(d.connect_with_connector_lazy(https_connector)))
```

Comment:
Could this be a `map`?

Reply:
Seems to work, i've committed the change as requested